README.md

```diff
@@ -72,6 +72,21 @@ GET example.com/api/v1/blogs/1
 GET example.com/api/v1/tech/blogs/1
 ```
 
+To sort your results, use **sort** as a query parameter. **sort** parameter is format of `?sort[sort_order]=sort_field`. If `sort_order` is not specified, the default sort order `asc` will be used
+```
+# Sort results by position - from the first to the last (ascending)
+GET example.com/api/v1/blogs/?sort=position
+
+# Sort results by created_at - from the newest to the oldest (descending)
+GET example.com/api/v1/blogs/?sort[desc]=created_at
+
+# Sort results by position - from the first to the last. If there are records with the same position, sort by created_at - from the newest to the oldest
+GET example.com/api/v1/blogs/?sort[]=position&sort[desc]=created_at
```

Comment:
Looks like this won't be able to handle multi-column sorting properly. How about using something like `sort=position,-created_at` where a minus sign (`-`) denotes `desc` and an optional plus sign (`+`) denotes `asc`? It's pretty common nowadays.

Reply:
I've changed back to previous way (using colon-separated sort_field:sort_order)